src/wallet.ts

```diff
@@ -781,3 +782,11 @@ export function coinselect(
 
   return { selectedUnspents: unspents, change };
 }
+
+export function estimateSize(
```

Comment:
This seems almost true for bitcoin, not elements. Where rhese numbers came from?

Reply:
This estimation looks too much approximated to me. You can take a look at how we estimate the tx size in the [tdex-daemon](https://github.com/TDex-network/tdex-daemon/blob/40d05a41111056e377d3a1ce4002663f732e190f/pkg/wallet/util.go#L106) as a starting point.
Consider that that estimation needs to be improved since it makes lower estimations than the real size of the tx.. let's talk offline about this eventually.